src/DotnetWebApiBench/BenchmarkRunner.cs

```diff
@@ -201,7 +208,14 @@ private Process StartWebApiServer(int httpsPort, string connectionString)
         {
             ProcessStartInfo startInfo = new ProcessStartInfo();
             startInfo.FileName = "DotnetWebApiBench.Api.exe";
-            startInfo.Arguments = $"{httpsPort} /ConnectionStrings:Northwind=\"{connectionString}\"";
+            if (configuration["serverType"] != null && configuration["serverType"].Equals("SQLServer", StringComparison.InvariantCultureIgnoreCase))
+            { 
+                startInfo.Arguments = $"{httpsPort} /ConnectionStrings:Northwind=\"{connectionString}\" /serverType={configuration["serverType"]}"; 
```

Comment:
Lets stick with current configuration parameters format convention. So for example use db-type instead of serverType to directly indicate that we are referring to a database not to a www server for example - applies to all occurences of the `serverType` configuration parameter

Reply:
changes applied